modules/navier_stokes/doc/content/source/physics/PNSFVSolidHeatTransferPhysics.md

```diff
@@ -2,6 +2,39 @@
 
 !syntax description /Physics/NavierStokes/SolidHeatTransfer/PNSFVSolidHeatTransferPhysics
 
+## Equation
+
+This [Physics](Physics/index.md) object creates the kernels and boundary conditions to solve the heat transfer
+equation for the solid phase in a porous media flow problem.
+
+!equation
+\dfrac{\partial (1-\epsilon) \rho h_s}{\partial t} - \nabla \cdot (\kappa_s \nabla T_s) - \alpha (T_f - T_s) - (1-\epsilon) \dot{Q} = 0
+
+where:
+
+- $h_s$ is the solid specific enthalpy, computed from the specific heat $c_{ps}$ and the solid temperature
+- $\rho$ is the solid density
+- $\epsilon$ is the porosity
+- $T_s$ is the solid temperature
+- $T_f$ is the fluid temperature
+- $k_s$ the solid thermal conductivity
+- $(1-\epsilon) Q$ is the source term, corresponding to energy deposited directly in the solid phase
+- $\alpha$ is the ambient convection volumetric heat transfer coefficient
+
+The enthalpy is used in lieu of $\rho c_p T$ to be able to model solids with temperature dependent
+specific heat.
+
+The kernels created for this equation are:
```

Comment:
```suggestion
The kernels potentially created for this equation are:
```
or just further clarify in some way that a user will not get both an isotropic and anisotropic diffusion kernel